content/en/bits_ai/bits_ai_sre/investigate_issues.md

```diff
@@ -0,0 +1,143 @@
+
+#### APM latency
+
+<!-- {{< callout url="http://datadoghq.com/product-preview/bits-ai-sre-pilot-features" >}}
+Bits AI SRE investigations from APM latency graphs and APM Watchdog stories are in Preview. Click <strong>Request Access</strong> to join the Preview program.
+{{< /callout >}} -->
+
+{{< callout url="#" btn_hidden="true" >}}
+Bits AI SRE investigations from APM latency graphs and APM Watchdog stories are in Preview.
+{{< /callout >}}
+
+##### APM latency graphs on service pages
+
+1. In Datadog, navigate to [APM][1] and open the service or resource page you want to investigate. Next to the latency graph, click **Investigate**.
+1. Click and drag your cursor over the point plot visualization to make a rectangular selection over a region that shows unusual latency to seed the analysis. Initial diagnostics on the latency issue appear, including the observed user impact, anomalous tags contributing to the issue, and recent changes. For more information, see [APM Investigator][2].
+1. Click **Investigate with Bits AI SRE** to run a deeper investigation. 
+
+##### APM latency Watchdog stories
+
+On a Watchdog APM latency story, click **Investigate with Bits AI SRE**. 
+
+### Enable automatic investigations
+
+In addition to manual investigations, you can configure Bits to run automatically when a monitor transitions to the alert state:
+
+#### Option 1: Bits AI SRE Monitors list
+1. Go to [**Bits AI SRE** > **Monitors** > **Supported**][5].
+1. Toggle **Auto-Investigate** on for a single monitor, or bulk-edit multiple monitors by selecting multiple monitors, then clicking **Auto-Investigate All**.
+
+#### Option 2: Configure automatic investigations for a single monitor
+1. Open the monitor's status page and click **Edit**.
+1. Scroll to **Configure notifications & automations** and toggle **Investigate with Bits AI SRE**.
+
+<div class="alert alert-info"><ul><li>Enabling automatic investigations using the Datadog API or Terraform is not supported.</li><li>An investigation initiates when a monitor transitions to the alert state.</li><li>Transitions to the warn or no data state, <a href="/monitors/notify/#renotify">renotifications</a>, and test notifications do not trigger automatic investigations.
+</li></ul></div>
+
+### Supported monitors
+
+Bits is able to run investigations on the following monitor types:
+  - Metric
+  - Anomaly
+  - Forecast
+  - Integration
+  - Outlier
+  - Logs
+  - APM (`APM Metrics` type only; `Trace Analytics` is not supported)
+  - Synthetics (API tests only)
+
+#### Best practices: Add investigation context to your monitors {#best-practices}
+Think of onboarding Bits as you would a new teammate: the more context you provide, the better it can investigate.
+
+1. **Include Datadog telemetry links**: Add at least one helpful telemetry link in the monitor message. Think about the first place you'd normally look in Datadog when this monitor triggers. It could be a link to any of the following:
```

Comment:
```suggestion
- **Include Datadog telemetry links**: Add at least one helpful telemetry link in the monitor message. Think about the first place you'd normally look in Datadog when this monitor triggers. It could be a link to any of the following:
```

Feel like this list should be unordered